pkg/install/resource/tm_cluster_role_binding.go

```diff
@@ -0,0 +1,56 @@
+package resource
+
+import (
+	"context"
+
+	rbac "k8s.io/api/rbac/v1"
+
+	"github.com/datawire/ambassador/pkg/kates"
+	"github.com/telepresenceio/telepresence/v2/pkg/install"
+)
+
+type tmClusterRoleBinding int
+
+var TrafficManagerClusterRoleBinding Instance = tmClusterRoleBinding(0)
+
+func (ri tmClusterRoleBinding) roleBinding() *kates.ClusterRoleBinding {
+	cr := new(kates.ClusterRoleBinding)
+	cr.TypeMeta = kates.TypeMeta{
+		Kind:       "ClusterRoleBinding",
+		APIVersion: "rbac.authorization.k8s.io/v1",
+	}
+	cr.ObjectMeta = kates.ObjectMeta{
+		Name: install.ManagerAppName,
+	}
+	return cr
+}
+
+func (ri tmClusterRoleBinding) Create(ctx context.Context) error {
+	clb := ri.roleBinding()
+	clb.Subjects = []rbac.Subject{
+		{
+			Kind:      "ServiceAccount",
+			Name:      "traffic-manager",
+			Namespace: "ambassador",
```

Comment:
This should not be hardcoded to "ambassador". A fix will arrive in subsequent commits.